content/etc/icingaweb2/config.ini

```diff
@@ -0,0 +1,10 @@
+[global]
+show_stacktraces = "1"
+config_backend = "db"
+config_resource = "icingaweb_db"
+module_path = "/usr/share/icingaweb2/modules:/etc/icingaweb2/modules"
```

Comment:
Agreed.  The RC version of Icinga Web 2 had the modules installed under /etc/icingaweb2/modules instead of /use/share/icingaweb2/modules.  I believe the RPM version still does that, but the DEB version was updated later to install the modules under /use/share/icingaweb2/modules.  Since this container is already in the wild and being used by users in Production, we'd need to make sure that any changes don't break the current set up. 

Perhaps install the module to their correct path under /usr/share/icingaweb2/modules then symlink from /etc/icingaweb2/modules to /usr/share/icingaweb2/modules ?